src/AppBundle/Tests/Unit/Validator/Constraints/RecaptchaValidatorTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
to keep it consistent with all other files the structure should look like this:

``` php
<?php

/*
 * top docblock
 */

declare(strict_types=1);

namespace Namespace;

class Class
{
}
```

(sry if my previous comment here was misleading)